bin/conjur-cli/commands/configuration/apply.rb

```diff
@@ -42,7 +42,7 @@ def call
       private
 
       def server_pid
-        cmd = "ps -ef | grep puma | grep -v grep | grep -v cluster | " \
+        cmd = "ps -ef | grep puma | grep -v grep | grep -v cluster | grep -v evoke | " \
```

Comment:
I think referring to the `evoke` name in Conjur seems like it should be a last resort, since this becomes an implicit dependency.

I would prefer us to pursue an alternative way to better identify the right Puma process(es) to terminate.

This is a helpful analysis, though! Thanks for digging into that. :)

Reply:
+1

Reply:
> I think referring to the evoke name in Conjur seems like it should be a last resort, since this becomes an implicit dependency.

that's a good point, thanks. feel free to close the PR or leave it open, whatever you think is best for when someone tries to properly fix this.